CONTRIBUTING.md

```diff
@@ -73,3 +73,27 @@ This is generally not necessary, but in case you need to manually publish a vers
 You can run `npm publish --dry-run` to see everything that _would_ happen during publish, without actually publishing to the npm registry.
 
 Note the branch is `v-next` for now. When we we merge this branch to `master`, these instructions should be updated.
+
+## Overriding source code previews in Storybook
+
+For most stories, we are able to generate a twig source code snippet for Storybook to display automatically. In some edge cases, the source code snippet cannot be generated automatically, so the JS that was passed to `<Story>` is shown instead. In those cases, you can manually pass the source code to `<Story>`:
```

Comment:
Do we know what those edge cases are? If so it might be nice to spell them out here so future devs can know whether their use case is one of those edge cases or not.

Reply:
As far as I know, the only case is if you use useEffect or another hook. But I don't know for sure if there could be other cases where the story function can't be called from the transformSource function

Reply:
I'll make a note about hooks causing it